js/views/modals/About/BTCTicker.js

```diff
@@ -0,0 +1,51 @@
+import loadTemplate from '../../../utils/loadTemplate';
+import baseVw from '../../baseVw';
+import { getExchangeRate } from '../../../utils/currency';
+
+
+const RATE_EXPIRY_S = '300';
+
+export default class extends baseVw {
+  constructor(options = {}) {
+    super({
+      className: 'aboutBTCTicker pureFlex alwaysFirst',
+      ...options,
+    });
+  }
+
+  getCurrentPrice() {
+    return getExchangeRate('USD');
```

Comment:
You would want to pass in `app.settings.get('localCurrency')` so it's displayed in the user's local currency.